internal/database/online/sqlite/store.go

```diff
@@ -47,3 +48,26 @@ func (db *DB) Purge(ctx context.Context, revisionID int) error {
 func (db *DB) Push(ctx context.Context, opt online.PushOpt) error {
 	panic("Implement me!")
 }
+
+func (db *DB) PrepareStreamTable(ctx context.Context, opt online.PrepareStreamTableOpt) error {
```

Comment:
implementations seems pretty similar for mysql, postgres, sqlite, can you put it to `sqlutil`?

Reply:
done